app/src/main/java/net/squanchy/navigation/HomeActivity.java

```diff
@@ -92,6 +96,24 @@ private void setupBottomNavigation(InterceptingBottomNavigationView bottomNaviga
         );
     }
 
+    private BottomNavigationSection getSelectedSectionOrDefault(Optional<Bundle> savedInstanceState) {
+        int selectedPageIndex = savedInstanceState.or(new Bundle())
+                .getInt(STATE_KEY_SELECTED_PAGE_INDEX, BottomNavigationSection.SCHEDULE.ordinal());
+        return BottomNavigationSection.values()[selectedPageIndex];
+    }
+
+    private void selectInitialPage(BottomNavigationSection section) {
```

Comment:
I'd extract a common method (for this and `selectPage`) so that if we need to change something while selecting a page we won't have 2 things to change